### DISCUSSION TITLE
Installing extensions inside docker build

### DISCUSSION THREAD

**Discussion by damienrj** [gitpod-io/openvscode-server#461] [Q&A]
Hello, 
I want to install the extensions at build time since we don't have access through our firewall when we deploy our containers.    Are they are instructions? I have been trying to do `/home/.openvscode-server/bin/openvscode-server --install-extension ms-python.python-2022.16.1.vsix` after downloading but am getting an unclear error.

**Comment by jeanp413** [comment]
@damienrj maybe you are missing `--start-server`, if you are running the server in your docker file [like in the official image](https://github.com/gitpod-io/openvscode-releases/blob/main/Dockerfile) then it would be like this
```
ENTRYPOINT [ "/bin/sh", "-c", "exec ${OPENVSCODE_SERVER_ROOT}/bin/openvscode-server --host 0.0.0.0 --without-connection-token --install-extension ms-python.python-2022.16.1.vsix --start-server \"${@}\"", "--" ]

```
